### ISSUE TITLE
Right OLED is just blank?

### ISSUE THREAD

**Issue by zawnk** [dnlbauer/splitkeyboard#6]
My nice!views finally arrived and I've soldered them on. The left one immediately worked without me having to do anything, just turning the right half back on. The right side just stays 'off' or at least doesn't display anything and just stays white. The OLED itself isn't faulty, because it works on the left side just fine. Am I missing any configuration for ZMK or anything to get the right one to work? Do I have to do anything special to get it to work? The jumper pads are all soldered, I've double checked that.

Thanks for the help!

**Comment by dnlbauer** [comment]
the oleds should not be affected by the firmware to my knowledge. maybe there is an issue with the soldering that leads to a short circuit and therefore the nice nano is not turning on? do you have a multimeter to measure if there are any shorts between the pins?

**Comment by zawnk** [comment]
Mhhh, I just tested to make sure, tried:
- socket on the PCB top and bottom to see if the socket mount is working fine and that worked
- on the bottom side of the PCB I checked if there's a short between the pins so I checked if any of them are connected but that seemed fine (no connection between any pins)
- checked the top of the OLED with its pins that I've soldered on but they also seem to work fine (but the OLED works on the other half just fine so I was 99% sure this isn't the problem)

Also turned the right half on and off and the reset button, but that didn't help either, the screen is still off

**Comment by zawnk** [comment]
I've tested the soldering on the display and its sockets again and the continuity seems fine with no shorts. 

But I've used the multi to check the voltage between VCC and GND and on the left side it's 3.2-3.3V and the right side shows 0 (even on the bottom side of the PCB). The jumpers are soldered on the bottom so that shouldn't be it. Are there any other pins that I could've missed?

**Comment by zawnk** [comment]
Turns out I needed to use `&ext_power EP_TOG` a few times and just toggle the external power supply for a bit for a reason that's not clear to me  :D

So if someone else has this problem: despite what the docs say, just bind that functionality and toggle the power and and off for a bit. At first, it only turned my left side off as well and nothing changed on the right, but after about a minute the right side turned on as well seemingly randomly. Now it seeems to work.

A bit quirky but I'll take it :)
